.pre-commit-config.yaml

```diff
@@ -109,7 +109,7 @@ repos:
       - id: types
         name: types
         files: ^(frontend|packages/js)/.*$
-        entry: bash -c 'pnpm run prepare:nuxt && pnpm run -r types'
```

Comment:
I don't really get why a lot of `language: system` hooks have followed the `bash -c ''` notation so I support this change.

Reply:
I don't know that either, but it works without it :)